MatrixSDK/Crypto/KeyBackup/MXKeyBackup.m

```diff
@@ -328,8 +338,9 @@ - (void)sendKeyBackup
         }
         else
         {
-            // Come back to the ready state so that we will retry on the next received key
+            // Retry a bit later
             self.state = MXKeyBackupStateReadyToBackUp;
```

Comment:
Is there a risk of infinite retry in case of no network issue, or other permanent error?

Reply:
Yes but no because the `MXHTTPClient` will wait that the network is back. 
In case of other kind of error (like timeout on bad network connectivity), yes, we can loop but with a delay between 0 and 10s and it seems to be the best to do in this case.